fleet/lib/ccs-influxdb/values.yaml

```diff
@@ -0,0 +1,49 @@
+---
+ingress:
+  enabled: true
+  tls: true
+  secretName: influxdb-tls
+  hostname: cameras-influxdb.${ .ClusterLabels.site }.lsst.org
```

Comment:
Perhaps `influxdb.ccs.<site>.lsst.org`?

Reply:
everything was changed to ccs-(deployment).site.lsst.org this was a leftover I didn't see earlier thanks!